package/src/skia/NativeSetup.ts

```diff
@@ -0,0 +1,18 @@
+import { NativeModules } from "react-native";
+
+if (global.SkiaApi == null) {
+  // Initialize RN Skia
+  const SkiaModule = NativeModules.RNSkia;
+  if (SkiaModule == null || typeof SkiaModule.install !== "function") {
+    throw new Error(
+      "Native RNSkia Module cannot be found! Make sure you correctly " +
+        "installed native dependencies and rebuilt your app."
+    );
+  }
+  const result = SkiaModule.install();
+  if (result !== true) {
+    throw new Error(
+      `Native Skia Module failed to correctly install JSI Bindings! Result: ${result}`
+    );
+  }
```

Comment:
You can replace it with :
```
if (!global.SkiaApi) {
  // Initialize RN Skia
  const SkiaModule = NativeModules.RNSkia;
  if (!SkiaModule || typeof SkiaModule.install !== "function") {
    throw new Error(
      "Native RNSkia Module cannot be found! Make sure you correctly " +
        "installed native dependencies and rebuilt your app."
    );
  }
  const result = SkiaModule.install();
  if (!result) {
    throw new Error(
      `Native Skia Module failed to correctly install JSI Bindings! Result: ${result}`
    );
  }
}
```

Reply:
lol I know that I _can_, but I won't because those implicit boolean checks are hard to read.